config/optional/field.field.group.microsite.lgms_footer_banner.yml

```diff
@@ -5,8 +5,6 @@ dependencies:
     - field.storage.group.lgms_footer_banner
     - group.type.microsite
     - media.type.image
-  module:
-    - localgov_media
```

Comment:
The footer no longer needs localgov_media? Probably good :)

But I think it was for image field and cropping formats.